functest/main_test.go

```diff
@@ -319,47 +327,22 @@ func setup() error {
 	}
 	fmt.Println("[ setup ] insolar was successfully builded")
 
-	err = generateRootMemberKeys()
-	if err != nil {
-		return errors.Wrap(err, "[ setup ] could't generate root member keys: ")
-	}
-	fmt.Println("[ setup ] root member keys successfully generated")
-
-	err = loadRootKeys()
-	if err != nil {
-		return errors.Wrap(err, "[ setup ] could't load root keys: ")
-	}
-	fmt.Println("[ setup ] root keys successfully loaded")
-
-	err = generateDiscoveryNodesKeys()
-	if err != nil {
-		return errors.Wrap(err, "[ setup ] could't generate discovery node keys: ")
-	}
-	fmt.Println("[ setup ] discovery nodes keys successfully generated")
-
-	err = buildInsolard()
-	if err != nil {
-		return errors.Wrap(err, "[ setup ] could't build insolard: ")
-	}
-	fmt.Println("[ setup ] insolard was successfully builded")
-
-	err = startGenesis()
-	if err != nil {
-		return errors.Wrap(err, "[ setup ] could't start genesis: ")
-	}
-	fmt.Println("[ setup ] genesis was successfully started")
-
 	err = startInsgorund()
```

Comment:
In launchnet.sh we already starting insgorund

Reply:
ok